src/pages/Schedule.js

```diff
@@ -0,0 +1,314 @@
+                        </div>
+                    </td>
+                    <td css={`padding-left: 1vw`}>
+                        <EventDescription description={description}/>
+                        {btext && <div><br/><Button href={blink}>{btext}</Button></div>}
+                    </td>
+                </tr>
+                </tbody>
+            </table>
+        </div>
+    )
+}
+
+const EventDescription = (props) => {
+    var linkRegex = /https?:\/\/.*$/;
+
+    return(
+        <Text css={`white-space:pre-line`}>
+            {(props.description).map((str, i) => {
+                const isLink = linkRegex.test(str);
+
+                if (isLink) {
+                    const link = str.match(linkRegex);
+                    const linkIdx = str.indexOf(link);
+                    const linkName = str.substring(0, (linkIdx));                    
+                    return (<Link key={i} href={link}>&nbsp;{linkName}</Link>)                         
+                } else {
+                    return (<span key={i}>{str}</span>)                   
+                }
+            })}
+        </Text>
+    )
+}
+
+const Timeline = () => {
+    return (
+        <div>
+            <EventGroup>
+                <Subtitle fontColor={GREEN}>Registration</Subtitle>
+                <Subtitle fontSize="0.6rem">Check Your Eligibility</Subtitle>
+                <EventNoDate>
+                    <Text>All students enrolled at any institution around the world are invited to 
+                        compete in the PCV Prize.<br/>
+                        Teams of up to 5 will pitch their early stage climate startup concepts to 
+                        industry leaders and venture capitalists.<br/>
+    ​                    Team eligibility varies by competition track. <br/>
+                    ​</Text>
+                    <Button>Official Rules</Button><br/>
+                </EventNoDate>
+                <Subtitle fontSize="0.6rem">Pick a Track</Subtitle>
+                <EventNoDate>
+                    <Text><b>Collegiate Tracks</b><br/>
+                            1. Waste & Circularity <br/>
+                            2. Energy & Transportation​ <br/>
+                            3. Food & Agriculture <br/>
+                            4. Wildcard <br/>
+                            ​<br/> 
+                            <b>High School Tracks</b><br/>
+                            1. Spark Teen High School Track <br/></Text><br/>
+                    <Button>Find Your Track</Button><br/>
+                </EventNoDate>
+                <EventWithDate>
+                <Subtitle fontSize="0.6rem">Individual Registration</Subtitle>
+                <EventWithDateInfo date="MAR 29 11:59 PM EST"
+                                description={['Register individually and start building your team through our online PCV Prize Discord community and stay up to date with our latest news.']}
```

Comment:
Prefer formatting the description as children of this component and using JSX instead of writing a list 🤔

Reply:
This happens in every event but I will not be commenting on every event.